drivers/display/uc81xx_regs.h

```diff
@@ -106,26 +113,8 @@
 #define UC8179_CDI_DDX1				BIT(1)
 #define UC8179_CDI_DDX0				BIT(0)
 
-struct uc81xx_tres {
-	uint16_t hres;
-	uint16_t vres;
-} __packed;
-
-BUILD_ASSERT(sizeof(struct uc81xx_tres) == 4);
-
-struct uc81xx_ptl {
-	uint16_t hrst;
-	uint16_t hred;
-	uint16_t vrst;
-	uint16_t vred;
-	uint8_t flags;
-} __packed;
-
-BUILD_ASSERT(sizeof(struct uc81xx_ptl) == 9);
-
```

Comment:
Instead, please rename it to uc81xx_tres16, uc81xx_ptl16 and add uc81xx_tres8 and uc81xx_ptl8.

Reply:
done